gtsam/symbolic/tests/symbolicExampleGraphs.h

```diff
@@ -26,38 +26,56 @@
 #include <gtsam/symbolic/SymbolicBayesTree.h>
 #include <gtsam/inference/Ordering.h>
 #include <gtsam/inference/Symbol.h>
-#include <boost/assign/list_of.hpp>
 
 namespace gtsam {
   namespace {
 
-    const SymbolicFactorGraph simpleTestGraph1 = boost::assign::list_of
-      (boost::make_shared<SymbolicFactor>(0,1))
-      (boost::make_shared<SymbolicFactor>(0,2))
-      (boost::make_shared<SymbolicFactor>(1,4))
-      (boost::make_shared<SymbolicFactor>(2,4))
-      (boost::make_shared<SymbolicFactor>(3,4));
-
-    const SymbolicBayesNet simpleTestGraph1BayesNet = boost::assign::list_of
-      (boost::make_shared<SymbolicConditional>(0,1,2))
-      (boost::make_shared<SymbolicConditional>(1,2,4))
-      (boost::make_shared<SymbolicConditional>(2,4))
-      (boost::make_shared<SymbolicConditional>(3,4))
-      (boost::make_shared<SymbolicConditional>(4));
-
-    const SymbolicFactorGraph simpleTestGraph2 = boost::assign::list_of
-      (boost::make_shared<SymbolicFactor>(0,1))
-      (boost::make_shared<SymbolicFactor>(0,2))
-      (boost::make_shared<SymbolicFactor>(1,3))
-      (boost::make_shared<SymbolicFactor>(1,4))
-      (boost::make_shared<SymbolicFactor>(2,3))
-      (boost::make_shared<SymbolicFactor>(4,5));
+    // A small helper class to replace Boost's `list_of` function.
```

Comment:
I think this is not needed because you can always use an initializer list...

Reply:
I invite you to do all the edits :-)